tdrs-backend/tdpservice/email/templates/access-request-count.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html' %}
```

Comment:
@elipe17 this email looks really good. i was able to test it by changing the crontab settings in DAC

![1336p1](https://user-images.githubusercontent.com/63075587/234703271-26ee74b8-63e0-4d7e-9497-719aa5d8c543.PNG)